integration-tests/src/test/java/org/web3j/ens/EnsIT.java

```diff
@@ -12,19 +12,21 @@
  */
 package org.web3j.ens;
 
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+import org.web3j.EVMTest;
 import org.web3j.protocol.Web3j;
-import org.web3j.protocol.http.HttpService;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@Disabled
```

Comment:
Should this be kept disabled?

Reply:
Yes, when this runs the response I get is that the node is not synched